plugins/volumeFilter_test.go

```diff
@@ -287,36 +287,37 @@ func TestVolumeFilterFn(t *testing.T) {
 	accountIDs := []string{}
 
 	for _, k := range testCases {
-		t.Run(k.name, func(t *testing.T) {
-			// exactly one of the two cap values must be set
-			if k.sellBaseCapInBase == nil && k.sellBaseCapInQuote == nil {
-				assert.Fail(t, "either one of the two cap values must be set")
-				return
-			}
-
-			if k.sellBaseCapInBase != nil && k.sellBaseCapInQuote != nil {
-				assert.Fail(t, "both of the cap values cannot be set")
-				return
-			}
+		for _, action := range []queries.DailyVolumeAction{queries.DailyVolumeActionSell} {
```

Comment:
PR submission tip:
it's best not to add changes like this in follow-on PR changes since it introduces a large amount of red and green lines. As a reviewer, this makes reviewing this PR 10x more complex, since I have to ensure nothing has changed in here and whatever has changed makes sense, etc.
I say this now, because I'm hoping to start leaving similar tips on PRs that will allow us to increase our PR efficiency (such as the comment about not resolving review comments). Hopefully this will reduce the PR fatigue for both of us, and happy to get similar thoughts and tips from you as well :)